.github/workflows/main.yml

```diff
@@ -32,15 +35,11 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
-    - name: install missing or frequently-updated deps
-      shell: bash
-      run: |
-        cpanm Mail::IMAPTalk # in image, but fetch latest!
     - name: setup git safe directory
       shell: bash
       run: git config --global --add safe.directory /__w/cyrus-imapd/cyrus-imapd
     - name: fetch upstream release tags
-      if: ${{ github.repository != 'cyrusimap/cyrus-imapd' }}
+      # if: ${{ github.repository != 'cyrusimap/cyrus-imapd' }}
```

Comment:
Is this intentional? We only need to fetch upstream tags if we are on someone elses branch IIRC?

Reply:
I can't recall. 😕  I've removed the commenting-out and pushed.